supervisor/__main__.py

```diff
@@ -2,12 +2,15 @@
 import asyncio
 from concurrent.futures import ThreadPoolExecutor
 import logging
+from pathlib import Path
 import sys
 
 from supervisor import bootstrap
 
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
+CONTAINER_OS_HEALTH = Path("/os/overlay/supervisor.health")
```

Comment:
This puts the device onto a non-volatile storage. I think it would be better if we have it somewhere volatile.

How about creating a volatile directory shared between OS and the supervisor, e.g. bind mounting host `/run/supervisor/` to supervisor `/run/supervisor`. Then we can use `/run/supervisor/health`

Reply:
`health` sounds like it should stay there, how about `startup-marker`.

Reply:
that is a good idea. The path is already used for the watchdog but we can /run/supervisor:/run/os